Formula/aspectj.rb

```diff
@@ -0,0 +1,66 @@
+class Aspectj < Formula
+  desc "AspectJ: aspect-oriented programming for Java"
+  homepage "https://eclipse.org/aspectj/"
+  url "https://www.eclipse.org/downloads/download.php?file=/tools/aspectj/aspectj-1.8.9.jar"
+  sha256 "bab38539cbe6e932410a104c21e18f05b773c78001f8800993da29d718d31a1b"
+
+  bottle :unneeded
+
+  depends_on :java
+
+  def install
+    system "java", "-jar", "aspectj-#{version}.jar", "-to", "."
+    libexec.install Dir["lib/*.jar"]
+    doc.install Dir["doc/*"]
+    (bin/"ajc").write <<-EOS.undent
+      #!/bin/sh
+      set -e
+      java -cp "#{libexec}/aspectjtools.jar:$JAVA_HOME/lib/tools.jar:$CLASSPATH" -Xmx64M org.aspectj.tools.ajc.Main "$@"
+    EOS
+    (bin/"aj").write <<-EOS.undent
+      #!/bin/sh
+      set -e
+      java -classpath "#{libexec}/aspectjweaver.jar" "-Djava.system.class.loader=org.aspectj.weaver.loadtime.WeavingURLClassLoader" "-Daj.class.path=$ASPECTPATH:$CLASSPATH" "-Daj.aspect.path=$ASPECTPATH" "$@"
+    EOS
+    (bin/"aj5").write <<-EOS.undent
+      #!/bin/sh
+      set -e
+      java -classpath "#{libexec}/aspectjweaver.jar:$CLASSPATH" "-javaagent:$ASPECTJ_HOME/lib/aspectjweaver.jar" "$@"
+    EOS
+    (bin/"ajbrowser").write <<-EOS.undent
+      #!/bin/sh
+      set -e
+      java -classpath "#{libexec}/aspectjtools.jar:$JAVA_HOME/lib/tools.jar:$CLASSPATH" -Xmx64M org.aspectj.tools.ajbrowser.Main "$@"
+    EOS
+    (bin/"ajdoc").write <<-EOS.undent
+      #!/bin/sh
+      set -e
+      java -classpath "#{libexec}/aspectjtools.jar:$JAVA_HOME/lib/tools.jar:$CLASSPATH" -Xmx64M org.aspectj.tools.ajdoc.Main "$@"
+    EOS
```

Comment:
Yeah. Not just to reduce maintenance on the Homebrew side, but so all AspectJ users have consistent commands. Lack of full deployment support is a common annoyance with Java tools.